admins.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import urllib
+
+import requests
+
+from client import get_token, get_github3_client
+
+if __name__ == '__main__':
+    # only using this for members_urlt
+    gh = get_github3_client()
```

Comment:
Doesn't this send a login request? Do we need to fully instantiate a gh client instance just to get to the URL template?